### ISSUE TITLE
A bit of assistance - Crash Ventura 13.6.9

### ISSUE THREAD

**Issue by Hebhansen** [dstuecken/sshfs-gui#25]
I am not a friend with terminal, hence, I have installed sshfs-gui for remote access to my server. FTP client is registered and all seems good. When entering my credentials in GUI it crashes.

I can see these are dependencies:

Mac OS X 10.5+ and at least one of the utilities listed below is required.

[MacFUSE](http://code.google.com/p/macfuse/) ([OSXFUSE](http://osxfuse.github.com/) in MacFUSE compatilibity mode should also work)
[SSHFS console utility by Takayama Fumihiko at pqrs.org](http://pqrs.org/macosx/sshfs/index.html)

MacFuse says no longer maintained.
Console by Fumihiko does not exist.

Basically I am trying to remote edit over SFTP in Visual Code Editor. I am completely new to the editor and this GUI etc etc. So is there another simpler way to access via FTP, that's what I am looking for.

Is GUI supported? is there a work around to crashes?

Thx Henrik
